### DISCUSSION TITLE
Problems constructing octokit/rest after upgrade to v17

### DISCUSSION THREAD

**Discussion by aril-spetalen** [octokit/octokit.js#1740] [General] [answered]
Hello,

I've upgraded Octokit from 16.42.2 to 17.9.2.

A first smelly change I did was ignoring this typecheck, but at least, I think it is according to the guidelines: 

`// tslint:disable-next-line: no-var-requires`
`const { OctoKit } = require('@octokit/rest');`

Later, I call 
`new OctoKit({ auth: `token ${token}`});`

The script fails with the exception: "TypeError: OctoKit is not a constructor".
Can't see the difference from what's suggested here: https://github.com/octokit/rest.js/issues/1207 .

Any ideas what I do wrong here, and what could bring me onwards?

Kind regards, 
Aril

**Comment by aril-spetalen** [comment, marked as answer]
Nevermind - I didn't know what I was doing. OctoKit -> Octokit in require statement was my problem.

**Reply by gr2m** [reply]
Yap, thanks for sharing the answer 👍
